src/lib.rs

```diff
@@ -208,11 +212,13 @@ pub fn find_sysroot() -> PathBuf {
 
 /// Checks the current rust version and channel.
 /// Exits if the minimum requirement is not met.
-pub fn check_rust_version() {
+pub fn check_rust_version(input: &Input) {
     let rustc_version = rustc_version::version_meta().unwrap();
 
-    if rustc_version.channel > Channel::Nightly {
-        eprintln!("cargo-3ds requires a nightly rustc version.");
+    // If the channel isn't nightly, we can't make use of the required unstable tools.
+    // However, `cargo 3ds new` doesn't have these requirements.
+    if rustc_version.channel > Channel::Nightly && input.cmd.should_compile() {
```

Comment:
Seems like a reasonable relax of the requiement 👍

Reply:
Indeed, it felt weird needing to pass `+nightly` before even getting into the project folder.